src/main/java/com/github/steveice10/mc/protocol/packet/ingame/clientbound/scoreboard/ClientboundSetPlayerTeamPacket.java

```diff
@@ -9,8 +9,8 @@
 import io.netty.buffer.ByteBuf;
 import lombok.*;
 import net.kyori.adventure.text.Component;
+import org.jetbrains.annotations.Nullable;
 
-import javax.annotation.Nullable;
 import java.io.IOException;
```

Comment:
Is checker framework available right now?

Reply:
No. The code also uses jetbrains annotation a lot. My PR just drops javax annotation for good.